add ability to specify client identifiers in rate limit rules (#122)
signalsciences/terraform-provider-sigsci: docs/guides/3.0.0.md

## Upgrading to version 3

**Rate limit rules now support non-IP client identifiers**

> To add support for non-IP client identifiers within rate limit rules w ehad to convert them from `TypeMap` to `TypeSet`.

Author: ```suggestion
To add support for non-IP client identifiers within rate limit rules we had to convert them from `TypeMap` to `TypeSet`.
```
